### DISCUSSION TITLE
Since update, plant pots are now person object?

### DISCUSSION THREAD

**Discussion by mcdent** [blakeblackshear/frigate#2880] [Ask A Question]
Hi,
I noticed after I updated that some plant pots on the garden are regularly being detected as a person. Its not happening at night at all but fairly consistently in the daylight with a little sun.
I have not changed any settings in my config file and am wondering if I should just move the plant pots or make a change to the config? 😁
![snapshot-1645878194 880875-aw8543](https://user-images.githubusercontent.com/8769856/155900401-9f73b405-a8de-4318-85f9-00186129815a.jpg)

**Comment by blakeblackshear** [comment]
I would set a min_area filter for person. The model in the latest version is the same as previous versions, so nothing has changed in the accuracy of the detections. It's other features surfacing false positives that would have existed previously under the right conditions.
